apollo-ios/Sources/Apollo/ClientAwarenessMetadata.swift

```diff
@@ -11,12 +11,12 @@ public struct ClientAwarenessMetadata: Sendable {
   /// The name of the application. This value is sent for the header "apollographql-client-name".
   ///
   /// Defaults to `nil`.
-  public var clientApplicationName: String?
+  public let clientApplicationName: String?
 
   /// The version of the application. This value is sent for the header "apollographql-client-version".
   ///
   /// Defaults to `nil`.
-  public var clientApplicationVersion: String?
+  public let clientApplicationVersion: String?
 
   /// Determines if the Apollo iOS library name and version should be sent with the telemetry data.
   ///
```

Comment:
_This comment is not for this line but down on [L28](https://github.com/apollographql/apollo-ios-dev/pull/681/files#diff-37d41f6d8707f6c8ae93505134ae6d66628227b45a1ca58df249eb01a64fb59dR28) instead._

`includeApolloLibraryAwareness` should probably be have `private(set)` added to the definition; set once via the initializer is all that's needed. Alternatively a `let` with no default.

Reply:
That's fair. I'll change that in the next PR that I make.